WordPress/Tests/PHP/YodaConditionsUnitTest.inc

```diff
@@ -84,3 +84,7 @@ return 0 == $foo; // OK
 return $foo == 0; // Bad
 
 if ( (int) $a['interval'] === (int) $b['interval'] ) {} // OK
+
+if ( $GLOBALS['wpdb']->num_rows === 0 ) {} // Bad
+
+if ( $true == strtolower( $check ) ) {} // Bad
```

Comment:
I guess I misspoke, I actually meant comparison. :blush: There was a test for the function on the left but not on the right.